api-client/src/extrinsic/offline_extrinsic.rs

```diff
@@ -42,7 +42,7 @@ impl<T: Config, Client> Api<T, Client> {
 	) -> UncheckedExtrinsic<ExtrinsicAddress<T>, Call, Signature<T>, TxExtension<T>> {
 		match self.signer() {
 			Some(signer) => compose_extrinsic_offline!(signer, call, self.extrinsic_params(nonce)),
-			None => UncheckedExtrinsic { preamble: Preamble::Bare(5), function: call },
```

Comment:
This is the only "real" change in this PR. 
This PR added a new field `encoded_call` to the `UncheckedExtrinsic` type: https://github.com/paritytech/polkadot-sdk/pull/10390